Formula/dpkg.rb

```diff
@@ -16,13 +16,19 @@ class Dpkg < Formula
 
   depends_on "pkg-config" => :build
   depends_on "gnu-tar"
+  depends_on "gpatch"
   depends_on "xz" # For LZMA
 
   def install
     # We need to specify a recent gnutar, otherwise various dpkg C programs will
     # use the system "tar", which will fail because it lacks certain switches.
     ENV["TAR"] = Formula["gnu-tar"].opt_bin/"gtar"
 
+    # Since 1.18.24 dpkg mandates the use of GNU patch to prevent occurrences
+    # of the CVE-2017-8283 vulnerability.
+    # http://www.openwall.com/lists/oss-security/2017/04/20/2
+    ENV["PATCH"] = Formula["gpatch"].opt_bin/"patch"
```

Comment:
Not unless I missed something obvious when I checked last night. This is the format upstream uses.